packages/amplify-category-auth/src/index.js

```diff
@@ -394,6 +395,36 @@ const executeAmplifyHeadlessCommand = async (context, headlessPayload) => {
         .then(getUpdateAuthRequestAdaptor(context.amplify.getProjectConfig().frontend, context.updatingAuth.requiredAttributes))
         .then(getUpdateAuthHandler(context));
       return;
+    case 'import':
+      if (projectHasAuth(context)) {
+        return;
+      }
+      await validateImportAuthRequest(headlessPayload);
+
+      const srvcMetaData = supportedServices.Cognito;
+      const { provider } = srvcMetaData;
+      const providerPlugin = context.amplify.getPluginInstance(context, provider);
+      const cognito = await providerPlugin.createCognitoUserPoolService(context);
+      const identity = await providerPlugin.createIdentityPoolService(context);
+      const {
+        userPoolId,
+        identityPoolId,
+        nativeClientId,
+        webClientId,
+      } = JSON.parse(headlessPayload);
```

Comment:
use `JSONUtilities.parse` from `amplify-cli-core`

Reply:
done. I had an issue exporting the `JSONUtitlities` class from `lib/index` file when running the unit tests. So imported from `lib/index/jsonUtilities` instead and that worked. I am guessing the `export *` was causing an issue inside the test execution since other export * also didn't work when run inside of the tests.